mp/game/neo/cfg/user_default.scr

```diff
@@ -0,0 +1,119 @@
+// User options script
+//
+// Format:
+//  Version [float]
+//  Options description followed by 
+//  Options defaults
+//
+// Option description syntax:
+//
+//  "cvar" { "Prompt" { type [ type info ] } { default } }
+//
+//  type = 
+//   BOOL   (a yes/no toggle)
+//   STRING
+//   NUMBER
+//   LIST
+//
+// type info:
+// BOOL                 no type info
+// NUMBER       min max range, use -1 -1 for no limits
+// STRING       no type info
+// LIST          delimited list of options value pairs
+//
+//
+// default depends on type
+// BOOL is "0" or "1"
+// NUMBER is "value"
+// STRING is "value"
+// LIST is "index", where index "0" is the first element of the list
+
+
+// Half-Life User Info Configuration Layout Script (stores last settings chosen, too)
+// File generated:  Tue Jul 13 17:43:58 AM
+//
+//
+// Cvar	-	Setting
+
+VERSION 1.0
+
+DESCRIPTION INFO_OPTIONS
+{
+	"neo_name"
+	{
+		"Fake Name"
+		{ STRING }
+		{ "" }
+	}
+	"cl_onlysteamnick"
+	{
+		"Only use Steam names"
+		{ BOOL }
+		{ "0" }
+	}
+	"neo_fov"
+	{
+		"FOV (60 - 110)"
+		{ NUMBER 60 110 } // Alternatively SLIDER, but can't have both?
+		{ "90" }
```

Comment:
One more thing I have to point out: as per the currently open PR #390 , we're likely going to change both the min FOV and default FOV values to 75. So that should probably be reflected here. Otherwise looks good.